rs/sns/testing/src/sns.rs

```diff
@@ -420,13 +629,13 @@ pub mod pocket_ic {
     // 3) sns - SNS canisters.
     // 4) canister_id - ID of the canister that will be upgraded.
     // 5) upgrade_arg - Arguments that will be passed to the canister during the upgrade.
-    pub async fn upgrade_sns_controlled_test_canister(
+    pub async fn propose_sns_controlled_test_canister_upgrade(
```

Comment:
Why "test canister"? This function can upgrade any sns-controlled canister, right? As in, it doesn't make assumptions about what canister it's upgrading, does it?

Reply:
That's a fair point
In the beginning, the basic scenario was based on the "test canister," hence the name. However, currently, the basic scenario supports arbitrary canister upgrades.
However, the WASM module to which the canister will be upgraded is hardcoded at the moment, while we should allow user-defined canister upgrades. Changing this should be simple

Reply:
Already did in https://github.com/dfinity/ic/pull/4705/commits/961329f2ad0c4327c7d96c948d36748895bcc16b:)